geode-serialization/src/main/java/org/apache/geode/internal/serialization/filter/SanctionedSerializablesFilterPattern.java

```diff
@@ -98,6 +98,7 @@ private static StringJoiner dependenciesPattern() {
 
         // jar deployment
         .add("com.sun.proxy.$Proxy*")
+        .add("jdk.proxy*")
```

Comment:
Will `jdk.proxy*.$Proxy*` work?

Reply:
Unfortunately, no. The filter matcher pattern is not a regular expression. See the rules for the filter pattern here: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/ObjectInputFilter.Config.html#createFilter(java.lang.String)

Reply:
Yeah. Seems like the number in the class name could be up to `jdk.proxy6`. Adding `jdk.proxy*` is good.